packages/material/scss/popup/_theme.scss

```diff
@@ -1 +1,17 @@
 @import "~@progress/kendo-theme-default/scss/popup/theme";
+
+@include exports( "list/theme/material") {
```

Comment:
These styles are not needed. The focused state is added in the material/dropdowns branch, and everything else is already handled by the popup theme.